### PR TITLE
add doxygen in getg2ir.f, Part of #55

### PR DESCRIPTION
**Pull request by Hang-Lei-NOAA**
update for progress, it is shown on
https://hang-lei-noaa.github.io/NCEPLIBS/NCEPLIBS-g2/html/index.html

Part of #55

### CODE REVIEW

**Comment by GeorgeGayno-NOAA** [comment]
@Hang-Lei-NOAA Add the reference to the issue in the description. If you reference it in the PR title, it won't appear in the issue.

**Comment by Hang-Lei-NOAA** [comment]
Thanks, George. 
I was trying to link the issue through the option in left panel. But the system says that no more than 10 pull request can be linked.

File: src/getg2ir.f [hunk 1]
```diff
@@ -1,74 +1,70 @@
...
-      USE RE_ALLOC          ! NEEDED FOR SUBROUTINE REALLOC
+C>    @note SUBPROGRAM CAN BE CALLED FROM A MULTIPROCESSING ENVIRONMENT.
+C>    DO NOT ENGAGE THE SAME LOGICAL UNIT FROM MORE THAN ONE PROCESSOR.
+C>
+C>    @author Mark Iredell @date 1995-10-31
+C>
+
+      SUBROUTINE GETG2IR(LUGB,MSK1,MSK2,MNUM,CBUF,NLEN,NNUM,NMESS,IRET)
+
+      USE RE_ALLOC          !< NEEDED FOR SUBROUTINE REALLOC
```
Comment (GeorgeGayno-NOAA) [inline src/getg2ir.f]: Since this is not a variable, I am not sure you need to add doxygen here (or on line 67).

**Comment by GeorgeGayno-NOAA** [comment]
@Hang-Lei-NOAA Most of the comments are in uppercase. I think we decided not to 'shout'.

File: src/getg2ir.f (hunk 1 above)
Comment (Hang-Lei-NOAA) [reply src/getg2ir.f]: I will change all, and change it to lower case.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Looks good.

**Pull request merged**